src/compiler/ruby_generator.cc

```diff
@@ -115,6 +115,50 @@ void PrintService(const ServiceDescriptor *service, const grpc::string &package,
 
 }  // namespace
 
+grpc::string SnakeCaseToCamelCase(grpc::string input) {
```

Comment:
You probably don't need this anymore, now that you have the other implementation.